RELEASE-NOTES.txt

```diff
@@ -2,7 +2,7 @@
 ------
 * [**] Add support for changing overlay color settings in Cover block
 * Add enter/exit animation in FloatingToolbar
-
+* Remove `Clear all settings` button from Image block settings
```

Comment:
Actually, we should add the [*] indicating it's not an "hi pri" change for test

.
```suggestion
* [*] Remove `Clear all settings` button from Image block settings
```